src/web/theme/ui/atoms/Colors/Colors.scss

```diff
@@ -21,11 +21,11 @@ $warning: #fd7737;
 $error: #ff0000;
 
 $backgroundColor: $white;
-$fontColor: #333333;
+$fontPrimaryColor: $brandPrimary;
+$fontSecondaryColor: $brandSecondary;
```

Comment:
For me, the main font color here is the `$brandSecondary`, the "accent" one is `$brandPrimary`. So I would have defined the font colors this way:
```
$fontColor: $brandSecondary;
$fontAccentColor: $brandPrimary;
```

Reply:
I understand ... but I would have done the opposite. with `$brand` and `$brandAccentColor` because primary to me mean 'important'